_notebooks/textbook.py

```diff
@@ -130,3 +144,22 @@ def clean_raw_text(self, disable_pysbd=False):
             if not disable_pysbd:
                 clean_text = "\n".join(seg.segment(clean_text))
             chapter.clean_text = clean_text
+    
```

Comment:
I think we need to remove the inline tqdm and just use it in the outside loops?

<img width="457" alt="image" src="https://user-images.githubusercontent.com/3250749/103843054-1f5a8880-50bd-11eb-9f7c-9f4fff6b7f26.png">

Reply:
Removed some of the extra tqdm love. xD